pelican/tests/output/custom/unbelievable.html

```diff
@@ -43,9 +43,9 @@ <h1 class="entry-title">
         </abbr>
 
         <address class="vcard author">
-                By <a class="url fn" href="./author/alexis-metaireau.html">Alexis Métaireau</a>
+                By                         <a class="url fn" href="./author/alexis-metaireau.html">Alexis Métaireau</a>
```

Comment:
this is definitely not caused by the pygments update. it was introduced via some change to the used theme but the tests never got updated.